### PR TITLE
Disable SUBMIT button when there is no network connection

### PR DESCRIPTION
**Pull request by bkonyi**
image

image

### CODE REVIEW

**Review by ferndot — DISMISSED** [review]
LGTM! Thanks

File: lib/src/ui/widgets/questions/step_finished_button.dart [hunk 1]
```diff
@@ -51,13 +53,22 @@ class StepFinishedButton extends StatelessWidget {
...
           alignment: isLastStep ? null : AlignmentDirectional.centerEnd,
           width: 400,
-          child: RaisedButton(
-            key: ValueKey('stepFinishedButton'),
-            onPressed: validated ? () => _handleOnPressed(context) : null,
-            child: Text(isLastStep
-                ? localizations.symptomReportSubmitButton
-                : localizations.symptomReportNextButton),
-          ),
+          child: ValueListenableBuilder<bool>(
```
Comment (ferndot) [inline lib/src/ui/widgets/questions/step_finished_button.dart:55]: 👍 great call using a listenable here!

**Comment by bkonyi** [comment]
> LGTM! Thanks

No problem! Looks like I can't merge since the CI had an infrastructure failure. I kicked the bot with a new commit updating a comment, but it looks like I'll need reapproval.

**Review by ferndot — APPROVED** [review]
Re-approved

**Pull request merged**